src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -176,6 +183,47 @@ class Storage extends React.Component {
         },
       },
 
+      // Bootable column - displayed only when editing disk
+      // Note: only one disk can be bootable at a time
+      {
+        header: {
+          label: msg.createVmStorageTableHeaderBootable(),
+          formatters: [
+            (...formatArgs) => {
+              const isEditing = Object.entries(this.state.editing).length > 0
+              return isEditing && headerFormatText(...formatArgs)
+            },
+          ],
+          props: {
+            style: {
+              width: '5%',
+            },
+          },
+        },
+        cell: {
+          formatters: [inlineEditFormatter],
```

Comment:
can we customize cell formatters here similar as it was done for header formatters? This would keep local(for this column only) logic into leaking to global scope (for all columns)

Reply:
I've tried so I know we can but I'm not sure if it's a better option. I found some disadvantages with such an approach and ... Sometimes it's good to have everything (related to one thing, for example cell formatters) in one place: you know where you find all the related stuff and you can change something easily for all of the columns at once. But yeah, this has also some disadvantages as you've mentioned above.

Reply:
Done. And it also helped me with fixing a potential issue regarding checking column's `label` :)